spec/controllers/course/forum/forums_controller_spec.rb

```diff
@@ -18,6 +18,16 @@
 
     before { sign_in(user) }
 
+    context 'with a non course user' do
```

Comment:
Let's keep this consistent with other specs and use `when`: 'when current user is not enrolled in the course'